test/regress/run_regression.py

```diff
@@ -370,18 +343,25 @@ def run_regression(unsat_cores, proofs, dump, use_skip_return_code,
            '--no-check-models' not in all_args and \
            '--debug-check-models' not in all_args:
             extra_command_line_args = ['--debug-check-models']
-        if unsat_cores and re.search(r'^(unsat|valid)$', expected_output):
-            if '--no-check-unsat-cores' not in all_args and \
+        if re.search(r'^(unsat|valid)$', expected_output):
+            if check_unsat_cores and \
+               '--no-check-unsat-cores' not in all_args and \
                '--check-unsat-cores' not in all_args and \
                '--incremental' not in all_args and \
                '--unconstrained-simp' not in all_args:
                 extra_command_line_args += ['--check-unsat-cores']
+            if check_proofs and \
+               '--no-check-proofs' not in all_args and \
+               '--check-proofs' not in all_args:
+                extra_command_line_args += ['--check-proofs']
         if '--no-check-abducts' not in all_args and \
-            '--check-abducts' not in all_args:
+            '--check-abducts' not in all_args and \
+            '(get-abduct)' in benchmark_content:
```

Comment:
```suggestion
            'get-abduct' in benchmark_content:
```
get-abduct has arguments, so I believe this won't catch anything.